lib/scholar/metrics/discounted_cumulative_gain.ex

```diff
@@ -0,0 +1,94 @@
+defmodule Scholar.Metrics.DiscountedCumulativeGain do
+  @moduledoc """
+  Discounted Cumulative Gain (DCG) is a measure of ranking quality.
```

Comment:
First paragraph is always a short summary.

```suggestion
  Discounted Cumulative Gain (DCG) is a measure of ranking quality.

```